solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -469,6 +469,13 @@ This is the `JMSType` header field if publishing/consuming to/from JMS.
 | Read/Write
 | The correlation ID.
 
+| solace_deliveryCount
+| Integer
+| Read
+| The number of times the message has been delivered.
+
+Note that, while the Delivery Count feature is in controlled availability, `Enable Client Delivery Count` must be enabled on the queue and a flow rebind or reconnect may be necessary after `Enable Client Delivery Count` is turned on.
```

Comment:
1. Don't think we should directly say that this is controlled availability feature. SCSt's development cycle isn't tied to JCSMP's or the broker's development cycles, so this will become untrue when delivery counts eventually goes GA. For what its worth, JCSMP doesn't mention that this is a controlled availability feature either.
2. Does this work if you were to [restart the binding](https://docs.spring.io/spring-cloud-stream/docs/3.2.1/reference/html/spring-cloud-stream.html#binding_visualization_control)? If so, then change `flow rebind` to `consumer bindings need to restarted`. Users don't have direct access to the flow state.
3. Similarly, flow `reconnect` should be changed to `queue egress restart (or a flow reconnect more specifically)`. Just to make it clear to operators the action they could take to trigger a flow reconnect on all consumers.

Reply:
Forget what I said about point 1. We should still say that this is a controlled availability feature.